### PR TITLE
Updating mingw readme

### PR DESCRIPTION
**Pull request by zyad137**
updating the mingw readme to be more detailed and help more in building thrive

### CODE REVIEW

File: mingw_setup/readme.txt [hunk 1]
```diff
@@ -131,13 +131,46 @@ file in the next step.
...
+(put it in the place you like)and inside it Right-click and choose git clone in URL write:
+
+https://github.com/Revolutionary-Games/Thrive.git
+
+and in Directory the place you want to put the project files in and then \new_name_for_file_here 
+(ex. D:\bulding thrive\Thrive)
+
+then click ok
+
+9.adding the project to system path
```
Comment (jjonj) [inline mingw_setup/readme.txt]: It is not really necessary to add the project or anything to system path (you can verify this by removing the stuff you added to system path and rebuilding) so we should remove this point 9

File: mingw_setup/readme.txt [hunk 2]
```diff
@@ -200,7 +235,8 @@ cross-compiling, but it's convenient for us, too.
 Note that the build/Thrive.exe will not work as it is not placed with the 
 necessary DLL files.
 
-* An ogre config will show up when you start Thrive. Selecting a non-0 value for FSAA anti aliasing may or may not prevent a current issue with flickering on windows.
+* An ogre config will show up when you start Thrive. Selecting a non-0 value for FSAA anti aliasing may or may not prevent a current
+issue with flickering on windows.
```
Comment (jjonj) [inline mingw_setup/readme.txt]: We fixed the flickering issue so you can remove that part if you like

**Comment by jjonj** [comment]
Some good changes to the readme in there, I added two comments above that I'd like you to look at!
Also the whole process of downloading mingw and building luajit that you went through is not necessary so we don't need to add that to the readme! (though there was a bug that gave you an error with missing -llua51 that should be fixed with https://github.com/Revolutionary-Games/Thrive/commit/a12c961e9722a3269ae23a263ce029a4a54141f6)

**Comment by zyad137** [comment]
@jjonj  done the edits you suggested : 
1-removed point 9
2-removed: 
* An ogre config will show up when you start Thrive. Selecting a non-0 value for FSAA anti aliasing may or may not prevent a current issue with flickering on windows.

**Comment by jjonj** [comment]
Looks good! Merging!

**Pull request merged**